.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -47,4 +47,15 @@ body:
       description: If needed add any other context about the problem here or include screenshots
       placeholder: Additional context, screenshots, log output, etc.
     validations:
-      required: true
+      required: false
```

Comment:
We should get this reviewed by a writer. I'll ping Brent :-)

Reply:
How about the following:

We change "Report and address a bug" header to "Fix a bug."

Label: I will provide a fix for a bug.
Description: To provide a fix for a bug, select this checkbox and then Get started. Thanks for contributing.

Reply:
@brentholtorf I updated the template with your suggestions. 

![image](https://github.com/microsoft/BCApps/assets/117829001/6a115c23-601e-417b-9aee-26476143cae9)